### PR TITLE
Update integration-AMP.yml

### PR DESCRIPTION
**Pull request by tzemel**
amp_delete_computers_isolation
amp_options_computers_isolation
amp_put_computers_isolation
amp_get_computers_isolation

## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/25617

## Description
amp_delete_computers_isolation
amp_options_computers_isolation
amp_put_computers_isolation
amp_get_computers_isolation
## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [x] NO
   
## Must have
- [X] Tests

### CODE REVIEW

**Comment by tzemel** [comment]
fixed version issue and not test was added

**Comment by guyfreund** [comment]
@kirbles19 this is an internal contribution (XSOAR supported), therefore your review is needed here.

**Comment by guyfreund** [comment]
@tzemel please add a release notes file, please see [RN Docs](https://xsoar.pan.dev/docs/integrations/changelog) for more information.

**Comment by tzemel** [comment]
add Change Log

**Comment by tzemel** [comment]
i have added change log

**Comment by guyfreund** [comment]
> i have added change log
> […](#)
> On Wed, Aug 19, 2020 at 12:27 PM Guy Freund ***@***.***> wrote: @tzemel <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_tzemel&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=8R3n0EZ_PzPfgBnAJ3KLxl9J_tzZRiVmdtDVunrJaLo&s=0RruRMBUli4bDWNMBMqrnb-WhtAwmnwY4PsRdlBGz_w&e=> please add a release notes file, please see RN Docs <https://urldefense.proofpoint.com/v2/url?u=https-3A__xsoar.pan.dev_docs_integrations_changelog&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=8R3n0EZ_PzPfgBnAJ3KLxl9J_tzZRiVmdtDVunrJaLo&s=z5lBPBjR48VCHyxNYrsKdP2iVB8HszGipmHzz7gFdxU&e=> for more information. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_demisto_content_pull_8247-23issuecomment-2D676024283&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=8R3n0EZ_PzPfgBnAJ3KLxl9J_tzZRiVmdtDVunrJaLo&s=zcexuAtnCV1lkmrlT05JXlL1l4P9PkwmAVHdHDBXbqs&e=>, or unsubscribe <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ANVKBF4VTES7VKWQICXVSZ3SBOLJTANCNFSM4PKORYYQ&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=8R3n0EZ_PzPfgBnAJ3KLxl9J_tzZRiVmdtDVunrJaLo&s=Yst9Ex2WsaUsKGAPAPzVl-PE3AR-kwlUKcrMQgvZcaY&e=> .

@tzemel you should use `demisto-sdk update-release-notes` command to update the release notes. The changelog file update is not needed anymore, please revert it.
You can check the documentation I've mentioned above for more info on the `demisto-sdk update-release-notes` command.

**Comment by tzemel** [comment]
I don't use git control I am doing everything from the  ui can you help?

**Comment by guyfreund** [comment]
@tzemel yeah sure, I do it for you.

File: Packs/AMP/Integrations/integration-AMP.yml [hunk 1]
```diff
@@ -114,6 +121,25 @@ script:
     var getURL = function(command, args) {
         var base = urlDict[command];
         switch (command) {
+
+            case 'amp_delete_computers_isolation':
+                base += '/' + args['connector_guid'] + '/isolation';
+                delete args['connector_guid'];
+                return base
```
Comment (guyfreund) [inline Packs/AMP/Integrations/integration-AMP.yml:128]: why is the return here needed? we return `base` at the end of the switch case statement.
```suggestion
```

**Review by guyfreund — CHANGES_REQUESTED** [review]
If you want to commit changes, first pull since I've made some changes.

**Comment by tzemel** [comment]
No I am good, @guyfreund
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_guyfreund&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=x9c6zwqx6vPBZH3rVIya9EElyMm3p1yXxq9cZGm8HfU&s=rl_NeJSfWT5pYGdWsesB2fIaybLcjO4QefaQaqGEj0k&e=>'s
want to check with the customer, the customer replied last week that
everything is fine with the enhancement and now I am waiting

**Comment by guyfreund** [comment]
@tzemel what are we waiting for?

**Comment by tzemel** [comment]
@guyfreund
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_guyfreund&d=DwMCaQ&c=V9IgWpI5PvzTw83UyHGVSoW3Uc1MFWe5J8PTfkrzVSo&r=-oDaSfHi_h7DL_NtMRfbvCb6c9W2WjpjR56xcU449Us&m=x9c6zwqx6vPBZH3rVIya9EElyMm3p1yXxq9cZGm8HfU&s=rl_NeJSfWT5pYGdWsesB2fIaybLcjO4QefaQaqGEj0k&e=>'s
as
requested you wanted confirmation from the customer (working session that
he won't provide) i have asked the customer to share the integration that i
implemented there,
i just want to compare the 2 integration to see that we have a match, i
think i did change something with the uri but i am not sure.
let's wait a few more days and wait for his response. i hope he will share
the integration

**Comment by tzemel** [comment]
Hi All,

i tired to get the integration from the customer, however the customer is
not responding we can push it

**Comment by guyfreund** [comment]
@tzemel as I said, we need the customer confirmation that all is working, we need to see it live or in a video recording.
We can't move on with it until then.

**Comment by tzemel** [comment]
guy you got the confirmation last week from the customer, video dont think
its possible i asked for the integration even that i didnt get.
if you want you can take it with @Kate Pang <kpang@paloaltonetworks.com>

**Comment by tzemel** [comment]
@ShahafBenYakir @GavrielFilippov @Kate-Pang 
Shahaf after repeated request to participate in a meeting to verify the new commands according to @guyfreund request unfortunately the customer didn't participate in any meeting.
i don't know what to do know, you can close the case, and we will lose the new commands

**Comment by guyfreund** [comment]
Closing as there's no answer from @Kate-Pang in https://github.com/demisto/etc/issues/25617
@tzemel please reopen when we have an instance to test this contribution's changes...

**Comment by ShahafBenYakir** [comment]
@omercnet FYI

**Review by guyfreund — CHANGES_REQUESTED** [review]
@tzemel we went on a demo with the customer, what's left to do here:
- for the `amp_options_computers_isolation` command there's no output in war room, please add one that will correspond to the [API docs](https://api-docs.amp.cisco.com/api_actions/details?api_action=OPTIONS+%2Fv1%2Fcomputers%2F%7B%3Aconnector_guid%7D%2Fisolation&api_host=api.eu.amp.cisco.com&api_resource=Endpoint+Isolation&api_version=v1)
- Fix the conflicts in the PR

**Comment by ShahafBenYakir** [comment]
@guyfreund What is left here?

**Review by guyfreund — APPROVED** [review]

**Pull request merged**